activities/codebase-stewardship/odoo-codebases.md

```diff
@@ -0,0 +1,90 @@
+---
+type: Guide
+explains: How we use Odoo for keeping track of the codebases we work with
+---
+
+# Tracking codebases in Odoo
+
+We use [Odoo](../tool-management/odoo.md) to keep track of the codebases we work with.
+We use the Projects app where we have set up a workflow that reflects [the codebase lifecycle](lifecycle.md).
+Each codebase is represented by a Task, and each lifecycle by a column.
+
+## Columns
+
+The project has more columns than what the lifecycle defines for clarity of how a codebase enters and exit the lifecycle.
+Below, all columns are explained together with the criteria we use to move codebases to another column, called gating criteria.
+
+### Research
+
+Codebases we just discovered, all codebases start here.
+To put it here it needs to:
+
+1. be openly licensed
+2. be used by, or being developed for, a public organization
+3. be developed by, or on behalf of, a public organization
+4. look like it implements public policy
+5. be accompanied with an explanation for why it is a compelling codebase for stewardship
+
+A codebase could go from Research to Backburner, Identify or Archive depending on what we learn about it.
+After leaving Research, a codebase will never come back here.
+
+### Backburner
+
+Backburner contains codebases that we still find interesting, but that we for some reason will not prioritze at the moment.
+It can be moved here from Research, Identify, Commit or Assess and can also be moved back to them when we prioritize them again.
+
+### Identify
+
+Codebases can be moved to Identify when at least two Foundation for Public Code staff members think these are interesting codebases for stewardship.
+A codebase can exit Identify towards Commit when codebase stewards have received Foundation approval for possible stewardship of the codebase.
+Otherwise codebases may move to Backburner or Archive at any time.
+
+### Commit
+
+Codebase stewards have started talking to current maintainers about possible stewardship.
+It must be clearly understood by the codebase community what stewardship would mean.
+The codebase can exit Commit towards Assess when the community makes a visible expression of commitment to aspire to stewardship, for instance in the `README` of a codebase.
+
+### Assess
+
+To enter Assess the codebase maintainers must agree to assessment towards the criteria in the Standard for Public Code.
+This agreement should be made with knowledge that we will publish the codebase in [the list of codebases we work with](https://publiccode.net/codebases/).
+A codebase is ready for the next stage when both the codebase community and the codebase stewards agree that the assessment is done.
+
+### Incubate
+
+When both the codebase community and the codebase stewards agree that a codebase is ready, it can enter Incubation.
```

Comment:
What additional commitment does the Foundation need from a community to move a codebase into Incubate after Assess, instead of Archive? I can easily imagine that an assessment showing lots of work/changes required to meet the Standard could trigger a round of reflection/planning/recommitting/resource-gathering in a codebase community. Does that need to be reflected in the entry criteria for Incubate?

Reply:
@ericherman clarification: this is a reflection of our current practice. When our current practice becomes more clear, we can create an issue and update this.